Modules/BenchPress.Azure/Public/Confirm-ContainerAppManagedEnv.ps1

```diff
@@ -0,0 +1,49 @@
+﻿# INLINE_SKIP
+using module ./../Classes/ConfirmResult.psm1
+
+. $PSScriptRoot/../Private/Connect-Account.ps1
+# end INLINE_SKIP
+
+function Confirm-ContainerAppManagedEnv {
+  <#
+    .SYNOPSIS
+      Confirms that a Container App Managed Environment exists.
+
+    .DESCRIPTION
+      The Confirm-AzBPContainerAppManagedEnv cmdlet gets a Managed Environment using the specified Managed Environment
+      and Resource Group names.
+
+    .PARAMETER ResourceGroupName
+      The name of the resource group. The name is case insensitive.
```

Comment:
NIT: 

Could we do no periods in the parameter names? That's how the other descriptions are

Reply:
What name are you referring to?

Reply:
Sorry wasn't that clear. I meant in the parameter descriptions
```suggestion
      The name of the resource group. The name is case insensitive
```

Reply:
I'll add this to the inconsistencies ticket: #293 . This needs to go the other direction, all of the descriptions should be complete sentences with correct punctuation.